cpp/game_main.h

```diff
@@ -89,8 +95,7 @@ class GameMain :
 	/**
 	 * map graphic id to gamedata graphic.
 	 */
-	std::unordered_map<int, gamedata::graphic *> graphics;
-
+	std::unordered_map<ssize_t, gamedata::graphic *> graphics;
 
```

Comment:
Is there a specific reason for using ssize_t as index type. I thought (s)size_t is only used for machine-width-integers for memory management.